src/test/java/com/bulkbuy/enterprise/EnterpriseApplicationTests.java

```diff
@@ -76,13 +76,13 @@ void confirmSavedOrder(){
         order.setCarrier(carrier);
         order.setOrderStatus(orderStatus);
 
-        orderService.Create(order);
+        orderService.create(order);
 
         List<Order> orders = orderService.getAllOrders();
-        for (Order ord:orders)
-        {
-            if (ord.getOrderId() == orderId && ord.getQuantity() == quantity && ord.getOrderDate().equals(orderDate))
-            {
+        for (Order ord:orders) {
```

Comment:
Open brace should appear at the end of the same line as the declaration
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3

Reply:
This point has been debated since before computers were invented.